.gitignore

```diff
@@ -1,5 +1,5 @@
-dist
-ext-dist
-node_modules
-bower_components
-.publish
+/dist
```

Comment:
I added the webclient/bower_components symlink back and needed it to not be ignored so I could stage and commit it.